core/templates/pages/blog-dashboard-page/blog-dashboard-page.module.ts

```diff
@@ -16,100 +16,47 @@
  * @fileoverview Module for the blog-dashboard page.
  */
 
-import { APP_INITIALIZER, NgModule, StaticProvider, DoBootstrap} from '@angular/core';
 import { MatTabsModule } from '@angular/material/tabs';
 import { MatMenuModule } from '@angular/material/menu';
 import { MatButtonToggleModule } from '@angular/material/button-toggle';
 import { MatTooltipModule } from '@angular/material/tooltip';
-import { BrowserModule, HAMMER_GESTURE_CONFIG } from '@angular/platform-browser';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { downgradeComponent } from '@angular/upgrade/static';
-import { HttpClientModule } from '@angular/common/http';
-import { HTTP_INTERCEPTORS } from '@angular/common/http';
-import { RequestInterceptor } from 'services/request-interceptor.service';
 import { SharedComponentsModule } from 'components/shared-component.module';
 import { RouterModule } from '@angular/router';
-import { APP_BASE_HREF } from '@angular/common';
-
-import { OppiaAngularRootComponent } from 'components/oppia-angular-root.component';
 import { BlogDashboardPageComponent } from 'pages/blog-dashboard-page/blog-dashboard-page.component';
-import { platformFeatureInitFactory, PlatformFeatureService } from 'services/platform-feature.service';
-import { ToastrModule } from 'ngx-toastr';
-import { MyHammerConfig, toastrConfig } from 'pages/oppia-root/app.module';
-import { SmartRouterModule } from 'hybrid-router-module-provider';
-import { AppErrorHandlerProvider } from 'pages/oppia-root/app-error-handler';
 import { SharedBlogComponentsModule } from 'pages/blog-dashboard-page/shared-blog-components.module';
+import { NgModule } from '@angular/core';
+import { ToastrModule } from 'ngx-toastr';
+import { toastrConfig } from 'pages/oppia-root/app.module';
+import { BlogDashboardPageRootComponent } from './blog-dashboard-page-root.component';
+import { BlogAuthorDetailsEditorComponent } from './modal-templates/author-detail-editor-modal.component';
 
 @NgModule({
   imports: [
-    BrowserModule,
-    HttpClientModule,
     SharedComponentsModule,
     SharedBlogComponentsModule,
-    ToastrModule.forRoot(toastrConfig),
     MatTabsModule,
     MatMenuModule,
     MatTooltipModule,
+    ToastrModule.forRoot(toastrConfig),
+    RouterModule.forChild([
+      {
+        path: '',
+        component: BlogDashboardPageRootComponent
+      }
+    ]),
     MatButtonToggleModule,
     // TODO(#13443): Remove smart router module provider once all pages are
     // migrated to angular router.
-    SmartRouterModule,
-    RouterModule.forRoot([]),
-    BrowserAnimationsModule
   ],
   declarations: [
     BlogDashboardPageComponent,
-    BlogAuthorDetailsEditorComponent
+    BlogAuthorDetailsEditorComponent,
```

Comment:
Do we need to remove `downgradeComponent` part from this component as per the steps in the issue?